packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
```

Comment:
> This is the default value which is defined here to note that...

I find this wording to be confusing. It's in most of the comments and I'm not sure what it means.

Wonder what value this phrase adds. Could we remove it?

```
/*
 It may be appropriate to disable this rule for older projects, such as when ...
*/

Reply:
OK I think this is the default value from Airbnb....


I wonder if we can reword as this:

```
/*
    This rule was copied from the Airbnb style guide without additional changes. We copy it here in order to add this comment:

    ...
*/
```

Reply:
I'm thinking I'll just remove the default value statements. They are repetitive and probably don't provide much value.